### ISSUE TITLE
Tables: Ensure headers and cells are properly associated

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2532]
**OCIO Violation:** Ensure headers and cells are properly associated

**OCIO Notes:** 
- This header cell does not have a scope attribute set to a text value of: colgroup
- This header cell does not have a scope attribute set to a text value of: colgroup; or alternatively an id attribute set to a non-null text value
- This header cell does not have a scope attribute set to a text value of: rowgroup
- This header cell does not have a scope attribute set to a text value of: rowgroup; or alternatively an id attribute set to a non-null text value

**OCIO priority:** 6.1

More info in [this sheet](https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g?e=IXABwa) or the [ocio/amp dashboard](https://departmentofinterior.levelaccess.us/login.php)

**Instances** (by note)
There are four pages/tables affected by this

1. This header cell does not have a scope attribute set to a text value of: colgroup
- [ ] https://revenuedata.doi.gov/how-revenue-works/native-american-revenue/ - stage of production
- [ ] https://revenuedata.doi.gov/ - source
- [ ] https://revenuedata.doi.gov/#main-content - source
- [ ] https://revenuedata.doi.gov/how-revenue-works/revenues - phase

2. This header cell does not have a scope attribute set to a text value of: colgroup; or alternatively an id attribute set to a non-null text value
- [ ] https://revenuedata.doi.gov/how-revenue-works/native-american-revenue/ - stage of production
- [ ] https://revenuedata.doi.gov/how-revenue-works/revenues - phase
- [ ] https://revenuedata.doi.gov/fact-sheet/ - source, recipient, revenue type
- [ ] https://revenuedata.doi.gov/#main-content – source
- [ ] https://revenuedata.doi.gov/ - source

3. This header cell does not have a scope attribute set to a text value of: rowgroup 
- [ ] https://revenuedata.doi.gov/how-revenue-works/native-american-revenue/ - resource

4. This header cell does not have a scope attribute set to a text value of: rowgroup; or alternatively an id attribute set to a non-null text value
- [ ] https://revenuedata.doi.gov/how-revenue-works/native-american-revenue/- resource

**Comment by eelzi-ONRR** [comment]
2024 report

Instances: 210
Occurrence: 8%
Severity: 10
Priority: 7.6

**Comment by lpgoldstein** [comment]
No longer relevant
